src/app/pages/organizations/onc-organizations/onc-organizations.jsx

```diff
@@ -199,30 +201,26 @@ function ChplOncOrganizations() {
                 )}
               { isEditing !== 'org' && orgType === 'acb'
                 && (
-                  <ChplUsers
-                    users={users}
-                    roles={roles}
-                    dispatch={handleDispatch}
-                  />
+                  <ChplUsers users={users} roles={roles} dispatch={handleDispatch} />
                 )}
             </>
           )}
         { !active?.id && !isCreating
          && (
          <Card>
            <CardContent>
-             <Typography>
-               ONC Organization maintenance
-             </Typography>
-             { hasAnyRole(['ROLE_ADMIN', 'ROLE_ONC'])
-               && (
-                 <Button
-                   onClick={() => setIsCreating(true)}
-                 >
-                   Create
-                 </Button>
-               )}
+             <Typography gutterBottom><strong>ONC Organization maintenance</strong></Typography>
            </CardContent>
+           <CardActions>
```

Comment:
This still needs the ` { hasAnyRole(['ROLE_ADMIN', 'ROLE_ONC'])` line, to make sure no ROLE_ACB users have a chance to try to create an ACB